javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,15 @@
+/* exported filterOutNulls */
+
+function filterOutNulls(values) {
+  var newArray = [];
+  var i = values.length;
+  while (i > 0) {
+    if (values[i - 1] !== null) {
+      newArray.unshift(values[i - 1]);
+      i--;
+    } else {
+      i--;
+    }
+  }
```

Comment:
Is this just for practice with `while` loops? If so, I would stick to keeping this a positive movement through the array, going backwards through the array adds an unnecessary level of complexity to the code.